modAionImpl/src/org/aion/zero/impl/sync/SyncStats.java

```diff
@@ -140,118 +122,67 @@ void update(long _blockNumber) {
      * @param type the type of request added
      */
     public void updateTotalRequestsToPeer(String nodeId, RequestType type) {
-        if (requestsEnabled) {
-            requestsLock.lock();
-            try {
-                RequestCounter current = requestsToPeers.get(nodeId);
-
-                if (current == null) {
-                    current = new RequestCounter(type);
-                    requestsToPeers.put(nodeId, current);
-                } else {
-                    switch (type) {
-                        case STATUS:
-                            current.incStatus();
-                            break;
-                        case HEADERS:
-                            current.incHeaders();
-                            break;
-                        case BODIES:
-                            current.incBodies();
-                            break;
-                    }
-                }
-            } finally {
-                requestsLock.unlock();
-            }
+        if (requestEnabled) {
+            requestTracker.updateTotalRequestsToPeer(nodeId, type);
+        }
+    }
+
+    @VisibleForTesting
+    Map<String, Float> getPercentageOfRequestsToPeers() {
```

Comment:
We could keep some of the Java Doc for this method

Reply:
added